data-processing/Dockerfile

```diff
@@ -20,6 +17,19 @@ RUN perlbrew init
 RUN perlbrew --notest install perl-5.22.4
 RUN perlbrew install-cpanm
 
+# Install LaTeX suite
+# This download will take an extremely long time to install, so we install it first
+RUN apt-get install -y wget
+RUN wget https://arxiv-web-static1.s3.amazonaws.com/semantic_scholar_20210412/arXivTeXLive2020.tar.gz
```

Comment:
I also have this copied to `s3://ai2-s2-scholarphi-pipeline-prod/arxiv-compiler/` for fun/backup, but getting it from S3 bucket requires AWS creds and cli. Any idea of a simple way to get it from our S3? If not, maybe it's okay to leave it this way?

Reply:
> for fun

🤣

Reply:
Kind of gross but we could parameterize the loc with a build arg for s3 path. Use the one that's hardcoded here as a default but allow us to use our own within our land of credentials (which all boxes building this should have).

Reply:
Ah I forgot that this is an open-sourced project. Changing it to S2's S3 will be problematic.